lib/reqTemplate.js

```diff
@@ -322,6 +322,16 @@ function replaceComplexTemplates(part, subSpec, globals) {
         return subSpec.map((elem) => replaceComplexTemplates(part, elem, globals));
     } else if (subSpec && subSpec.constructor === String || subSpec === '') {
         if (/\{[^}]+\}/.test(subSpec)) {
+            // Strip trailing newlines, so that we can use yaml multi-line
+            // syntax like this:
+            // someKey: >
+            //   {{ merge ({
+            //        "someKey": "foo",
+            //        "otherKey": "bar"
+            //      }, options }}
+            if (/^\{[\w\W]*\}\n$/m.test(subSpec)) {
```

Comment:
That test is for brackets anywhere though, and I wanted to be a bit more conservative about only targeting something that is very likely just an expression, and not a mix of string literal & templated bits.